README.md

```diff
@@ -7,6 +7,7 @@
 
 ## Environment variables
 Before starting the application or running tests, ensure that the correct environment variables have been set. A list of required environment variables are available in `sample.env`.
+The environment variable `NOTIFY_API_KEY` is required and needs to be a real test api key (not the whitelisted team one).
```

Comment:
I find this line a bit confusing. Why is this a test api key?

Reply:
I'll clarify, locally in .env you need to use a test one.